setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='testimony',
-    version='1.0.3',
+    version='1.0.4',
```

Comment:
Shouldn't make it 1.1.0? Since we are changing/adding the not need of `@test` tag?